.cloud/README.md

```diff
@@ -23,8 +23,8 @@ All those are direclty injected to the Cloud Run runtime through GCP Secrets.
 * [integration google analytics key variable](https://console.cloud.google.com/security/secret-manager/secret/contextual-help-api-google-analytics/versions?cloudshell=false&project=core-oss-integration)
 * [preprod app key variable](https://console.cloud.google.com/security/secret-manager/secret/contextual-help-api-key?cloudshell=false&project=core-oss-preproduction)
 * [preprod google analytics key variable](https://console.cloud.google.com/security/secret-manager/secret/contextual-help-api-google-analytics/versions?cloudshell=false&project=core-oss-preproduction)
-* [production app key variable SOON]()
-* [production google analytics key variable SOON]()
+* [production app key variable](https://console.cloud.google.com/security/secret-manager/secret/contextual-help-api-key/versions?cloudshell=false&project=core-oss-production)
+* [production google analytics key variable](https://console.cloud.google.com/security/secret-manager/secret/contextual-help-api-google-analytics/versions?cloudshell=false&project=core-oss-production)
```

Comment:
I know this URL cannot be accessed without being granted rights but is this OK to publish this publicly?